packages/vue3-vite/src/generators/component/schema.json

```diff
@@ -26,8 +26,7 @@
     "directory": {
       "type": "string",
       "description": "Create the component under this directory (can be nested).",
-      "alias": "d",
-      "default": "components"
```

Comment:
The default value of `--directory` option is removed in order to control the default directory for different type of projects (`apps/{name}/src/app/components` for apps, and `libs/{name}/src/lib` for libs).

It took me a while to realize that `--directory` option would actually affect the output of our generator, even if we didn't explicitly use it anywhere. I guess it somehow conflicts with the built-in `--directory` option of nx CLI. Moreover, it actually reads&uses the default value in `schema.json`.

For example, if we run `npx nx g nx-vue3-vite:component mycomp --project myapp` (no `--directory` is used):
1. We will not be able to see the `directory` property in the `options` object in generator, because it's not being used while running the command (so we can treat it as `undefined`, but the key does not exist).
2. The component will be generated in `apps/myapp/src/components` instead of `apps/myapp/src`, because the default value of `directory` is `components` in `schema.json`.
3. If we change the default value of `directory` in `schema.json` to something like `abc` and run the same command, the component will then be generated in `apps/myapp/src/abc`.

Reply:
Makes sense to me!